nbs/00_utils.ipynb

```diff
@@ -12,17 +12,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        if rng_reserve_size is not None:
</pre><p>Here, you are basically assiging values to config. Write it as a separate function so that we can reuse it for assigning <code>global_seed</code>.</p><p> </p><p>In addition, let's not check typing and value for now. It will raise error anyway. A better way of type checking is to refactor <code>Config</code> class using pydantic. See <a href="https://github.com/BirkhoffG/jax-relax/blob/master/relax/data_module.py#L79" target="_blank">DataModuleConfig</a> as an example. I don't want you to refactor <code>Config</code> in this PR. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Completed</p><p> </p>